netlify/netlify-plugin-nextjs
# chore: put default demo in demos/default

### Summary

- Move the default demo site: `/demo` => `/demos/default` (to allow multiple demo sites)
- Fix broken cypress test

Build settings have been updated to work with this pr

### Test plan

1. Visit deploy preview and see all is ok: https://619e32e2c1cae5402f223739--netlify-plugin-nextjs-demo.netlify.app/fr/
2. Visit deploy logs and see all is ok: https://app.netlify.com/sites/netlify-plugin-nextjs-demo/deploys/619e32e2c1cae5402f223739

### Relevant links (GitHub issues, Notion docs, etc.) or a picture of cute animal
#839 

### Standard checks:




---

🧪 Once merged, make sure to update the version if needed and that it was published correctly.
